Alcancia-io/alcancia
# Fix/authentication backend

## Summary
Made all the necessary changes to correctly get data from the backend again by adding missing queries, modifying existing ones, and adding a new MFA screen for login.
## Requirements
_list any dependencies necessary to review the changes_
- _eg. make sure you reaplace your `.env` with the latest version_

## Type of change

## How to test?
Run the app, create a new account as usual. Then go through the new sign in process that sends you to the MFA screen, wait for 1 minute to test the resend code button. After logging in everything else in the app should work as usual

## How has this been tested?
- [ ] Register new user
- [ ] Log in with MFA code
- [ ] Request multiple MFA codes for login
- [ ] Request multiple MFA codes for registration

## Screenshots
N/A

## Checklist (for the reviewer)
